src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -20,31 +20,39 @@ interface ArtistInfoProps
 
 export class ArtistInfo extends React.Component<ArtistInfoProps> {
   render() {
+    // FIXME: Fetch from relay
+    const props = {
+      href: "/artist/francesca-dimattio",
+      imageUrl: "https://picsum.photos/110/110/?random",
+      name: "Francesca DiMattio",
+      meta: "American, b. 1979",
+    }
+
     return (
-      <React.Fragment>
-        <Responsive>
-          {({ xs }) => {
-            return (
-              <Serif size={xs ? "5" : "8"} mb={2}>
-                {this.props.name}
-              </Serif>
-            )
-          }}
-        </Responsive>
-        <Box mb={1}>
-          <MarketInsights artist={artistResponse as any} />
-        </Box>
-        <Box mb={2}>
-          <SelectedExhibitions exhibitions={this.props.exhibitions} />
-        </Box>
-        <ArtistBio
-          bio={
-            {
-              biography_blurb: { text: this.props.bio, credit: "Gagosian" },
-            } as any
-          }
-        />
-      </React.Fragment>
+      <>
+        <StackableBorderBox p={2} flexDirection="column">
+          <EntityOverline {...props} />
+          <Spacer mb={1} />
+          <ArtistBio
+            maxChars={200}
+            bio={
+              {
+                biography_blurb: { text: this.props.bio, credit: "Gagosian" },
```

Comment:
This `Gagosian` should also come from props, right?

Reply:
Oops, will update! 🍝